### DISCUSSION TITLE
IRegionAware not available while implementing custom ScopedRegionNavigationContentLoader

### DISCUSSION THREAD

**Discussion by BabuLalSuthr** [PrismLibrary/Prism#3060] [WPF]
Hi Team,

I'm trying to implement ScopedRegionNavigationContentLoader as per https://www.pluralsight.com/courses/prism-mastering-tabcontrol course, but I noticed that IRegionAware is not available in my project
![image](https://github.com/PrismLibrary/Prism/assets/17385480/579acd8a-63c5-4e83-a352-9b6beae04ee8)

I noticed that IRegionAware is part of Region.Core and I've referenced Prism.Wpf in my project which also depends on Prism.Core.
![image](https://github.com/PrismLibrary/Prism/assets/17385480/1a0efc92-c8e6-464f-84ef-1cd00e5152ac)

![image](https://github.com/PrismLibrary/Prism/assets/17385480/280541c7-4cb1-4778-8f47-dda0ee5adf99)

**I'm using WPF with .Net Framework 4.8, Prism 8

Please let me know if I'm missing anything :)** 

Thanks

**Comment by BabuLalSuthr** [comment]
FYI - @brianlagunas

**Comment by dansiegel** [comment]
This issue isn't really surprising as IRegionAware doesn't exist in WPF in Prism 8.1. The move to core was done in Prism 9 and IRegionAware would be there with INavigationAware being provided in Prism.Wpf for backwards compatibility.
